library/src/main/java/com/wultra/android/sslpinning/integration/SSLPinningIntegration.kt

```diff
@@ -66,6 +75,16 @@ class SSLPinningIntegration {
             // use all trust managers after the provided (or our [SSLPinningX509TrustManager])
             val trustSslPinningCerts = arrayOf(sslPinningTrustManager, *originalTrustManagers)
             try {
+                if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
I guess the bound is incorrect. It's enbabled by default since API level 20. Not 22 (lollipop_mr1)

Reply:
Should be `if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP)`

https://developer.android.com/reference/javax/net/ssl/SSLSocket#default-configuration-for-different-android-versions

Reply:
Yop